tools/expotools/src/commands/NativeUnitTests.ts

```diff
@@ -6,8 +6,15 @@ import * as Directories from '../Directories';
 import { androidNativeUnitTests } from './AndroidNativeUnitTests';
 
 type PlatformName = 'android' | 'ios' | 'both';
+type TestType = 'local' | 'instrumented';
 
-async function thisAction({ platform }: { platform?: PlatformName }) {
+async function thisAction({
+  platform,
+  type,
```

Comment:
```suggestion
  type = 'local',
```

might help to specify a default value here and then make the type a required argument in `androidNativeUnitTests`